### DISCUSSION TITLE
courses/beginners-web-handbook/certificate

### DISCUSSION THREAD

**Discussion by giscus** [crwdu/lms-contents-js101#60] [Q&A]
# courses/beginners-web-handbook/certificate

바닐라코딩

https://lms.vanillacoding.co/courses/beginners-web-handbook/certificate

**Comment by junpil-TnT** [comment]
안준필 완강 인증합니다. 
https://junpil-tnt.github.io/

**Reply by Ken123777** [reply]
안녕하세요 준필님 수고하셨습니다 🙇🏻‍♂️ 진행하신 과제 파일 메일로 보내주시면 저와 1:1 체크인 예약 도와드릴게요!

**Comment by mjae404** [comment]
김민지 완강 인증합니다. :)
http://messenger.mjaecodecamp.xyz

**Comment by junpil-TnT** [comment]
-----Original Message-----

**Comment by HunKim-Dev** [comment]
김태헌 완강 인증합니다.
https://shushudf.github.io/

**Reply by Ken123777** [reply]
엇, 김태헌님. 고생하셨습니다. 🙇🏻 시작하신지 얼마 안된거 같은데 빠르게 하셨네요. 뒷 부분도 추가적으로 한번 살펴보고 학습해보세요! 저희 커뮤니티에 초대해드려야 하는데, 저희가 커뮤니티 정비 중이라, 며칠 안에 초대해드리겠습니다! 🙂
